core/src/main/java/com/kin/ecosystem/core/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -461,6 +468,15 @@ public boolean updateActiveAccount(int accountIndex) {
 		return false;
 	}
 
+	@Override
+	public void logout() {
+		removeRegistration(paymentRegistration);
```

Comment:
should we also check in login if user_id changed? If it did change we can throw an error - enforcing developer to explicitly call login before. Or we can logout implicitly?